### DISCUSSION TITLE
AAC audio on RTSP

### DISCUSSION THREAD

**Discussion by ttocsr** [gtxaspec/wz_mini_hacks#197] [General]
Is there any way to get AAC audio on the RTSP stream?
I currently use wyze bridge just for the purpose of getting AAC audio for RTSP.  wyze bridge is a great project, but its overkill just to get AAC audio.  The reason for AAC audio is Home Assistant.

**Comment by virmaior** [comment]
Similar discussion at https://github.com/gtxaspec/wz_mini_hacks/discussions/93

**Comment by ttocsr** [comment]
Yes it is similar. But I'm wondering if there is another way to avoid running an rtsp proxy and running ffmpeg on another box.
For me this is the one thing missing that lets me do everything directly from the camera.
I'm also using rtsp-simple-server/docker-wyze-bridge just to get AAC audio.

**Comment by gtxaspec** [comment]
We can convert the  audio to aac using ffmpeg, but we have no way to supply it to the v4l2rtspserver.  It only works with alsa devices.
